tests/transformers/test_model_wrapper.py

```diff
@@ -911,3 +913,86 @@ def test_dummy_wrapper_fails():
     input_features_wrong_order = input_features[1:] + [input_features[0]]
     with pytest.raises(ValueError):
         dummy.transform(df.loc[:, input_features_wrong_order])  # fail because wront number of columns
+
+
+class DummyOnlyDataFrame(BaseEstimator, TransformerMixin):
+    def __init__(self):
+        pass
+    
+    def fit(self, X, y=None):
+        if not isinstance(X, pd.DataFrame):
+            raise TypeError("only works for a DataFrame")
+            
+        return self
+    
+    def transform(self, X):
+        if not isinstance(X, pd.DataFrame):
+            raise TypeError("only works for a DataFrame")
+            
+        return X
+    
+
+class DummyNoDataFrame(BaseEstimator, TransformerMixin):
+    def __init__(self):
+        pass
+    
+    def fit(self, X, y=None):
+        if isinstance(X, pd.DataFrame):
+            raise TypeError("doesn't work on DatraFrame")
+            
+        return self
+    
+    def transform(self, X):
+        if isinstance(X, pd.DataFrame):
+            raise TypeError("doesn't work on DatraFrame")
+            
+        return X
+
+
+def test_AutoWrapper():
+    np.random.seed(123)
+    X = np.random.randn(100,10)
+    df = pd.DataFrame(X, columns=[f"NUMBER_{j}" for j in range(X.shape[1])])
+    df["not_a_number"] = "a"
+
+    model = AutoWrapper(TruncatedSVD(n_components=2, random_state=123))(columns_to_use=["NUMBER_"], regex_match=True)
+    Xres = model.fit_transform(df)
+    
+    assert isinstance(Xres, pd.DataFrame)
+    assert Xres.shape[0] == df.shape[0]
+    
+    
+    model = AutoWrapper(TruncatedSVD(n_components=2))(columns_to_use=["NUMBER_"], regex_match=True, column_prefix="SVD")
```

Comment:
Fix `random_seed` in test

Reply:
done : setted random_state in 2nd TruncatedSVD